pkgs/games/crawl/default.nix

```
@@ -52,7 +52,8 @@ stdenv.mkDerivation rec {
   makeFlags = [ "prefix=$(out)" "FORCE_CC=cc" "FORCE_CXX=c++" "HOSTCXX=c++"
                 "SAVEDIR=~/.crawl" "sqlite=${sqlite.dev}"
               ] ++ lib.optional tileMode "TILES=y"
-                ++ lib.optional enableSound "SOUND=y";
+                ++ lib.optional enableSound "SOUND=y"
+                ++ lib.optional stdenv.isDarwin "DATADIR=$(out)";
```

```suggestion
                ++ lib.optional stdenv.isDarwin "DATADIR=${placeholder ''out''}";
```

Rather useful nix buliltin. Can you do this for `prefix` too?